.gitignore

```diff
@@ -14,3 +14,6 @@ gfx/UltimateCataclysm/tall.png
 
 # BrownLikeBears built sprite sheets
 gfx/BrownLikeBears/tiles.png
+build/compose.py
+scratch/UltimateCataclysm/terrain/t_backboard.pdn
+tools/compose.py
```

Comment:
This seems like it doesn't belong in the repo's gitignore. Can you remove those lines and, perhaps, you might want to move them to your local system-based gitignore?